### ISSUE TITLE
Can you add support for multiple IDPs?

### ISSUE THREAD

**Issue by pbchase** [ctsit/shibboleth_and_table_auth#7]
From chris.wilhelmsen via the REDCap community:

```
...
I can confirm that I was able to install and configure both to display on the
Splash page and was able to use a Table based user as expected. I did run into
a hiccup on clicking the Shibboleth
link(https://{hostname}/Shibboleth.sso/Login?target=/), it sends me to a
Shibboleth error page that states: None of the configured SessionInitiators
handled the request. This is because this Shibboleth instance is configured
without an entityID in the <sso> declaration of my shibboleth2.xml so
Shibboleth does not know which entity to send the request to, if I manually
specify the entityID in a browser request, it works perfectly:
https://{hostname}/Shibboleth.sso/Login?target=/&entityID={IDP}. Your patch
does work out of the box if I configure Shibboleth to utilize the Discovery
Service or specify a entityID in shibboleth2.xml.

The reason I have this configured like this is to support multiple IDP's, so
this begs two questions:

Can the Shibboleth Link be Changed in some way to directly point to a specific
entityID/IDP? If I can change this, I am confident that this will work for us
at least for evaluation.

Can this Shibboleth+Table Based selection be expanded to support multiple
IDP's by allowing multiple Shibboleth tabs or images on the splash page? This
is obviously an extension of what is currently developed and may be a feature
I am only interested in so I certainly don't expect it at this time.

All and all great job on this, very exciting!
```

**Comment by pbchase** [comment]
I moved the first request for pointing to a specific IDP to its own issue, https://github.com/ctsit/shibboleth_and_table_auth/issues/9

**Comment by pbchase** [comment]
Addressed by PR #13
